mvtools/terraform-iac/ec2-userdata.tftpl

```diff
@@ -45,9 +45,11 @@ mkdir -p /root/code
 aws s3 cp s3://${s3_bucket}/code.zip /tmp/code.zip
 unzip /tmp/code.zip -d /root/code/
 aws s3 cp s3://${s3_bucket}/results.csv /root/code/results.csv 
-python3 /root/code/mvtools/python-survey-app/questionnaireresults.py
+python3 root/code/mvtools/python-survey-app/questionnaireresults.py > /root/code/clean_results.csv
```

Comment:
You don't need this line at all. It would actually overwrite the csv file with a print statement from questionaireresults.py. Ideally you need to save the file normally (like you do locally), and save the print output to .txt file.